docs/source/release_notes/release-notes.rst

```diff
@@ -48,6 +48,10 @@ develop
            that would otherwise require creating many new connections to Cassandra from the clients.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1402>`__)
 
+    *    - |fixed|
+         - Don't retry transactions when the locks are invalid. Previously, AtlasDB tried repeatedly to run a transaction when the external locks are already invalid.
```

Comment:
This was already fix in another PR, this one actually just amounted to a refactor.  I'll put up a PR to remove this.